### PR TITLE
Add possibility to add many handlers

### PR DESCRIPTION
**Pull request by Nek-**
This adds the capability to make a difference between different URIs called by the client.

It fixes https://github.com/Nekland/Woketo/issues/102 

Missing for now:
- [x] test
- [x] doc update

### CODE REVIEW

File: src/Server/WebSocketServer.php [hunk 1]
```diff
@@ -108,22 +118,27 @@ public function setMessageHandler($messageHandler)
                 throw new \InvalidArgumentException('The messageHandler must be a string representing a class.');
             }
         }
-        $this->messageHandler = $messageHandler;
+        $this->messageHandlers[$uri] = $messageHandler;
```
Comment (folliked) [inline src/Server/WebSocketServer.php]: I am not convinced that uri makes a good key. if they are 2 sames uri ?

Comment (Nek-) [inline src/Server/WebSocketServer.php]: It cannot !

Comment (folliked) [inline src/Server/WebSocketServer.php]: ok go :muscle:  @valanz hihi :rat:

Comment (valanz) [inline src/Server/WebSocketServer.php]: Too late 😿

**Pull request merged**